inc/plugin.class.php

```diff
@@ -106,6 +106,9 @@ static function getMenuContent() {
 
 
    static function getAdditionalMenuLinks() {
+      if (!static::canView()) {
```

Comment:
Not sure this one may cause a problem; sounds logic to me having this check there